### DISCUSSION TITLE
Run animation itself after loading page

### DISCUSSION THREAD

**Discussion by Firemanpl** [slint-ui/slint#3497] [General]
How can i animate for eg, image. I need to create animate width, height and then ? ... I want infinite animation scale with bezier curves on the page.

**Comment by Firemanpl** [comment]
and making custom animation functions based on blazier curves?

**Comment by ogoffart** [comment]
Maybe what you are looking for is the animation-tick() function: https://slint.dev/releases/1.2.0/docs/slint/src/language/builtins/functions#animation-tick-duration
